Gruntfile.js

```diff
@@ -176,33 +180,53 @@ module.exports = function( grunt ) {
 	} );
 
 	// Load grunt tasks from NPM packages
-	require( "load-grunt-tasks" )( grunt );
+	// Support: Node.js <4
+	// Don't load the eslint task in old Node.js, it won't parse.
```

Comment:
The chalk package.json file on master says it [still works](https://github.com/chalk/chalk/blob/d939a3f9bc70b9f6d265934047e4a191078180b4/package.json#L13) with node 0.10 and I don't see a `const` in it. Maybe one of its dependents?